stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -63,11 +64,14 @@
     private static final String LOGGING_ENDPOINT = "https://m.stripe.com/4";
 
     private static final String CHARSET = "UTF-8";
+    private static final String CUSTOMERS = "customers";
     private static final String TOKENS = "tokens";
     private static final String SOURCES = "sources";
     private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
     private static final SSLSocketFactory SSL_SOCKET_FACTORY = new StripeSSLSocketFactory();
 
+    static final String API_VERSION = "2017-06-05";
```

Comment:
So it's perfectly fine for the SDK and the user's backend to be on different API versions – the only exception to this rule is for the ephemeral key request, which is why we pass the SDK's API version to the user's backend. 

It's important for the SDK to be locked to an API version so that it doesn't receive unexpected responses that it can't parse, or pass unsupported parameters.